pkg/api/render_test.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"github.com/rebuy-de/kubernetes-deployment/pkg/interceptors"
+	"github.com/rebuy-de/kubernetes-deployment/pkg/testutil"
+	"testing"
+	"io/ioutil"
+	"path"
+)
+
+func readFile(t *testing.T, path string) string {
+	dat, err := ioutil.ReadFile(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return string(dat)
+}
+
+func TestDecode(t *testing.T) {
+	dir := "test-fixtures"
+	tcs := map[string]string{
+		"manifest-deployment.yaml": "",
```

Comment:
What do you think about this:

```
"manifest-deployment.yaml": readFile(t, path.Join(dir, fname)),
```

Reply:
yeah, that works too and is more succinct.

Reply:
done